download-utils/src/lib.rs

```diff
@@ -56,7 +56,7 @@ pub fn download_snapshot_archive(
     full_snapshot_archives_dir: &Path,
     incremental_snapshot_archives_dir: &Path,
     desired_snapshot_hash: (Slot, SnapshotHash),
-    snapshot_kind: SnapshotKind,
+    archive_kind: ArchiveKind,
```

Comment:
Same thing here:


```suggestion
    snapshot_kind: SnapshotArchiveKind,
```

(and the rest of this file too, please)